client/src/components/UploadCode.js

```diff
@@ -37,6 +38,7 @@ export default function AddCodeDialog({ dispatch, open, handleClose }) {
     };
 
     //SEND REQUEST AND close dialog
+    updateBalance(state.user.id, -codeReviewPrice, dispatch);
```

Comment:
there is a **-** before codeReviewPrice. not sure if that is intended

Reply:
There is only one function to change balance on BE. If you want to withdraw credits you specify summ with a '-' minus sign